src/CommunityToolkit/Xamarin.CommunityToolkit.Sample.Android/MainActivity.cs

```diff
@@ -17,14 +16,15 @@ protected override void OnCreate(Bundle savedInstanceState)
 			base.OnCreate(savedInstanceState);
 
 			global::Xamarin.Forms.Forms.SetFlags("CollectionView_Experimental");
-			Platform.Init(this, savedInstanceState);
+			Essentials.Platform.Init(this, savedInstanceState);
+			ToolkitPlatform.Init(this);
```

Comment:
Can we avoid initializing it somehow?
Maybe we should depend on CrossCurrentActivity plugin? @jfversluis @pictos @jsuarezruiz @sthewissen

Reply:
`CrossCurrentActivity` needs to be initialized as well... We still have access to the `Forms.Forms.Context` is obsolete, but will there until Maui. This on top of my head, after looking at the implementation I can have more (bad) ideas 😋

Reply:
This is required in the `NavigationExtensions.android.cs` as it uses `Platform.CreateRendererWithContext`.

I didn't have a good alternative that I could think of when implementing this. The Popup controls are quite different than standard renderers as it needs to invoke the `Platform.CreateRendererWithContext` API. Typically this isn't needed as most custom renderers are loaded in the context of a `ContentPage`. 

I looked at using the `[Obsolete]` API `Platform.CreateRenderer` for Android but didn't want new code to depend on deprecated APIs. I could see an argument for removing that deprecation flag in Xamarin.Forms 5, but I think that will cause more harm to the general ecosystem since using it with the correct `Context` is very important. 

I'm all ears for a better way to handle this as I don't like having the init API if we can avoid it. To add an additional mind to the mix, @PureWeen do you have any thoughts on this?

Reply:
@ahoefling when we perform the navigation, we already have the BasePopup instanciated? If so I think that we solve this more easily.

Reply:
Yes, we have an instance of the BasePopup, but there is no renderer associated with it unless we manually instantiate it. This is because the concept of Popups doesn't exist in Xamarin.Forms. I am using the CreateRenderer APIs to manually associate the shared object with it's native renderer.

Reply:
Well... I want to see if we can resolve this in the renderer area first and if we don't a way move to having an android lifecycle, to track the current Context.

Reply:
I'm really hoping someone will spot a thing I missed. It was very frustrating when I had to use this workaround 😢